.azure/templates/steps/system_test_general.yaml

```diff
@@ -107,7 +107,15 @@ jobs:
         publishJUnitResults: true
         testResultsFiles: '**/failsafe-reports/TEST-*.xml'
         goals: 'verify'
-        options: '-P${{ parameters.profile }} -DexcludedGroups=flaky,loadbalancer,networkpolicies,${{ parameters.excludedGroups }} -Dmaven.javadoc.skip=true -B -V -Dfailsafe.rerunFailingTestsCount=2 -Djunit.jupiter.execution.parallel.enabled=${{ parameters.run_parallel }} -Djunit.jupiter.execution.parallel.config.fixed.parallelism="${{ parameters.parallel }}"'
+        options: '
+          -P${{ parameters.profile }} 
+          -DexcludedGroups=flaky,loadbalancer,networkpolicies,${{ parameters.excludedGroups }} 
+          -Dmaven.javadoc.skip=true -B -V 
+          -Dfailsafe.rerunFailingTestsCount=2 
+          -Djunit.jupiter.execution.parallel.enabled=${{ parameters.run_parallel }} 
+          -Djunit.jupiter.execution.parallel.config.fixed.parallelism="${{ parameters.parallel }}" 
+          -Dskip.surefire.tests
+          '
```

Comment:
This looks super ugly. Can't you use YAMl block here? (https://yaml-multiline.info/)

Reply:
Sure, I was thinking about it but I decided to try this at first. Will change it.